datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -151,7 +151,7 @@ def add_item(self, item: DatasetItem, *args, **kwargs):
                 if related_images:
                     item_desc["related_images"] = related_images
             elif isinstance(item.media, MediaElement):
-                item_desc["media"] = {"path": item.media.path}
+                item_desc["image"] = {"path": getattr(item.media, "path", None)}
```

Comment:
Why is this key changed from `media` to `image`?

Reply:
Great catch! Fixed in https://github.com/openvinotoolkit/datumaro/pull/944/commits/3e84aebc6d0265037cb2e1945570272e2cf71c9d